lib/reverse_sentence.rb

```diff
@@ -1,6 +1,31 @@
 # A method to reverse the words in a sentence, in place.
-# Time complexity: ?
-# Space complexity: ?
+# Time complexity: O(m), m being the size of the string.I have two loops O(2m) but they are not nested. we drop the coeficient.
+# Space complexity: O(n), I created a new array that will handle the swapping of the words.
+
 def reverse_sentence(my_sentence)
```

Comment:
Given that it all depends on the size of the string if you're using `m` as the length of the string I would say O(m) for both.